### PR TITLE
Buffs Morphine by nerfing its sedative power

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

- Morphine now takes roughly double the amount of units before it forces sleep. 
	- Note, it can still cause sleep before this threshold, thanks to drowsiness RNG.
- Morphine now causes a mild amount of disgust. Never reaches enough to vomitlock.
- Doubles the addiction caused by Morphine

## Why It's Good For The Game

I have never seen Morphine used for its "intended use" 

Its description, `Causes drowsiness and eventually unconsciousness in high doses`, is a lie because the "high dose" it mentions is the smallest dose the codebase normally provides pills (5u). Any lower and it lasts too short to matter.

This means it's far more effective as a sedative than its intended use, which is lame, because there are even more effective sedatives. So why would you ever use it? 

By bumping up the sleep threshold to ~10 units, this pr intends to make Morphine a lot more usable - 5u pills will rarely cause sleep, and 10u pills will be a bit risky, but still "safe". Beyond 10 units it still guarantees sleep, so it can still be used in a syringe gun or in surgery.  

To offset its safety, it's now far more addictive, having the potential to cause Opiate addiction in just over 30 units. 

I also gave it a tiny bit of disgust, not enough to really matter, mostly for flavor.

TL;DR I want to be able to shoot a morphine autoinjector into my leg to escape from certain death, not to guarantee my death

## Changelog

:cl: Melbert
balance: Morphine's sedative power has been halved. It now takes ~10 units to guarantee sleep, up from ~5. 
balance: Morphine now causes mild disgust. 
balance: Morphine is now twice as addictive, causing addiction if you down over ~30 units without pause.
/:cl:

### CODE REVIEW

**Comment by Sadboysuss** [comment]
it would be nice if you reduced the metabolize speed as well so it can make you numb for longer periods of time without sleeping you
this way it could be used for surgeries more

**Comment by MrMelbert** [comment]
I tweaked all these numbers around the existing metabolism rate

**Comment by carlarctg** [comment]
the only morphine autoinjector i know of is in a spess ruin

**Comment by MrMelbert** [comment]
Yeah but what if I put some in cargo

**Review by ZephyrTFA — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
coolio

**Comment by Sakamoto4ka** [comment]
Sigmaface
